src/screens/Auth/RegisterStack/RegisterScreen/RegisterScreen.tsx

```diff
@@ -19,27 +19,26 @@ export default function RegisterScreen(props: Props) {
   const [email, setEmail] = useState('')
   const {navigate} = props.navigation
   const user = {
```

Comment:
make a variable like I mentioned above for props.navigation.state.params. It's a bit cleaner

Reply:
same error as I said for the other screen which I can't figure out

Reply:
sg nvm

Reply:
sorry I'm re-requesting review 100 times but I forgot to push last time